### ISSUE TITLE
how to train with 320*320

### ISSUE THREAD

**Issue by 11rocky** [AlexeyAB/darknet#1750]
if I want to train with 320*320, how to change the cfg file? Just change the height and width in the file? the pre-trained weights is also useful？

**Comment by AlexeyAB** [comment]
Just set `width=320 height=320` in your cfg file. Everything else is the same.

**Comment by 11rocky** [comment]
thanks! another question, if i train with 416*416, can i test with 320*320?

**Comment by AlexeyAB** [comment]
Yes, but you should follow this rule in this case: https://github.com/AlexeyAB/darknet#how-to-improve-object-detection

> General rule - your training dataset should include such a set of relative sizes of objects that you want to detect:
> 
> `train_network_width * train_obj_width / train_image_width ~= detection_network_width * detection_obj_width / detection_image_width`
